src/sentry/relay/config/metric_extraction.py

```diff
@@ -38,11 +39,16 @@ class MetricConditionalTaggingRule(TypedDict):
 _SATISFACTION_TARGET_METRICS = (
     "s:transactions/user@none",
     "d:transactions/duration@millisecond",
+    "d:transactions/measurements.lcp@millisecond",
```

Comment:
Out of curiosity, why are we adding this?

Reply:
I believe we need this to fix apdex for projects that calculate it based on LCP